src/clusterfuzz/_internal/metrics/monitor.py

```diff
@@ -117,10 +117,18 @@ def run(self):
           time_series.append(series)
 
           if len(time_series) == MAX_TIME_SERIES_PER_CALL:
+            time_series.sort(
+                key=
+                lambda ts: (ts.points[-1].interval.start_time.second, ts.points[-1].interval.start_time.nanosecond)  # pylint: disable=line-too-long
```

Comment:
Let's extract this to a helper function:

```py
def time_series_sort_key(ts):
  return ts.points[-1].interval.start_time
```

Reply:
Done.